site_scons/site_tools/t2t.py

```diff
@@ -14,7 +14,7 @@
 
 def txt2tags_actionFunc(target,source,env):
 	import txt2tags
-	txt2tags.exec_command_line([str(source[0])])
+	txt2tags.exec_command_line(["--outfile", str(source[0])[:-3] + "html", str(source[0])])

```

Comment:
Why this change?

Reply:
This was already on the branch, honestly I don't know why the change was made. It seems to do the same as the default (no args) command. I'll remove the change.